### PR TITLE
Daver2910 patch 1

### PR DESCRIPTION
**Pull request by Daver2910**
Added bootstrap and fixed the menu / logo bug in the mobile version.  I have issues viewing these changes on my local system. Can some please review the code. thanks

### CODE REVIEW

File: theses.css [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+    box-shadow: 0;
+    border: 3px solid #3498db;
+    color: #3498db;
+    outline: none;
+    padding: 13px 13px 13px 54px;
+}
+
+/* Icons ---------------------------------- */
+#feedback-name {
+    background-image: url(http://rexkirby.com/kirbyandson/images/name.svg);
```
Comment (agentydragon) [inline theses.css]: This should be hosted on our own domain. (Also below.)

Comment (agentydragon) [inline theses.css]: Also, I see a lot of opportunity for DRYing up all these icons (they all specify the same bg-size, bg-position, bg-repeat).

File: theses.css [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+
+.feedback-submit:hover .feedback-ease {
+    width: 100%;
+    background-color: white;
+}
+
+@media (max-width: 768px) {
+    #logo {
+        text-align: right;
+    }
```
Comment (agentydragon) [inline theses.css]: Missing '}'

File: theses.css [hunk 3]
```diff
@@ -0,0 +1,306 @@
...
+    color: #3498db;
+}
+
+.feedback-ease {
+    width: 0px;
+    height: 74px;
+    background-color: #fbfbfb;
+    -webkit-transition: .3s ease;
+    -moz-transition: .3s ease;
+    -o-transition: .3s ease;
```
Comment (agentydragon) [inline theses.css]: The styles above are missing -o-transition, -ms-transition. (Perhaps we could switch to a CSS generator that would handle prefixing for us?)

File: theses_base.html [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+{% load i18n %}
+{% load menu_tags %}
+{% load static %}
+{% load wagtailuserbar %}
+
+{% block extra_css %}
+    <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="{% static 'css/eacr.css' %}">
+    <link rel="stylesheet" type="text/css" href="{% static 'css/theses.css' %}">
+<<<<<<< HEAD
```
Comment (agentydragon) [inline theses_base.html]: These are leftover merge markers. Please make sure the merge is okay and remove them. (Here and below.)

File: theses_base.html [hunk 5]
```diff
@@ -0,0 +1,216 @@
...
+        <div class="row">
+            <div class="col-md-12">
+                {% block content %}
+                {% endblock %}
+            </div>
+        </div>
+        <hr class="mt-5"/>
+        <footer role="contentinfo">
+                    {% flat_menu "theses_footer" template="menus/footer_menu.html" %}
+            <p class="copyright">© 2017 <a href="">Efektivní Altruismus</a>.
```
Comment (agentydragon) [inline theses_base.html]: Please set href to http://efektivni-altruismus.cz

File: theses_base.html [hunk 6]
```diff
@@ -0,0 +1,216 @@
...
+                class="fa fa-google-plus"></i></a>
+
+        <!-- pinterest -->
+        <a class="pinterest"
+           href="https://pinterest.com/pin/create/bookmarklet/?url={{ page.full_url }}&is_video=false&description={{ page.title }}"
+           target="blank"><i class="fa fa-pinterest-p"></i></a>
+
+
+    </div>
+<<<<<<< HEAD
```
Comment (agentydragon) [inline theses_base.html]: More merge markers.

File: thesis_choose_help_page.html [hunk 7]
```diff
@@ -0,0 +1,45 @@
+<<<<<<< HEAD
```
Comment (agentydragon) [inline thesis_choose_help_page.html]: More merge markers

File: thesis_index_page.html [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+
+{% load wagtailcore_tags %}
+{% load wagtailuserbar %}
+{% load wagtailimages_tags %}
+
+
+{% block body_class %}template-thesisindex{% endblock %}
+
+{% block content %}
+<<<<<<< HEAD
```
Comment (agentydragon) [inline thesis_index_page.html]: More merge markers in this file.

File: thesis_search.html [hunk 9]
```diff
@@ -0,0 +1,166 @@
...
+<!-- Topics page layout 2/8/2 -->
+    <div class="col-md-2"></div> <!-- equal spacer on left -->
+        <div class="col-md-8">
+            {{ page.body | safe }}
+        </div>
+    <div class="col-md-2"></div> <!-- equal spacer on right -->
+</div> <!-- End of row -->
+
+
+<<<<<<< HEAD
```
Comment (agentydragon) [inline thesis_search.html]: Merge markers.

**Comment by agentydragon** [comment]
BTW, I'm surprised to see Dan's commits in your branch. This way, your branch might contain changes you weren't intending. Are Dan's commits already in master? If they are, can you try cherry-picking your commit on top of master?

**Comment by agentydragon** [comment]
Or perhaps you want to actually merge into this PR (https://github.com/ea-czech-republic/efektivnialtruismus.cz/pull/56/files), which has the 'redesign' branch, not into 'master'.

**Comment by Daver2910** [comment]
removing because of mistakes

**Pull request closed without merging**